File: Python/PracticeQue/List/Palindrom.py
a = []
num = int(input("Enter number of element in list : "))

for i in range(0,num):
    b = int(input(f"Enter {i} element of list : "))
    a.append(b)

rev = []
for j in range(len(a)-1,-1,-1):
    rev.append(a[j])

if rev == a:
    print("Your list is palindrom!")
else:
    print("Your list isn't palindrom!")

File: C/Palindrom.c
#include <stdio.h>

int main(){
    int rem,rev = 0,tem,n;

    printf("Enter your number : ");
    scanf("%d",&n);

    // store number in tem variable bcz we change n value
    tem = n;

    while(n != 0){
        rem = n % 10;
        rev = (rev * 10) + rem;
        n/=10;
    }

    if(tem == rev){
        printf("%d is palindrom number.!!",rev);
    }
    else{
        printf("%d isn't palindrom number.!!",rev);
    }
    return 0;
}